modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -429,7 +444,7 @@ cv::GArg IECallContext::packArg(const cv::GArg &arg) {
 
 struct IECallable {
     static const char *name() { return "IERequestCallable"; }
-    using Run = std::function<void(cv::gimpl::ie::IECompiled&, std::shared_ptr<IECallContext>)>;
+    using Run = std::function<void(std::shared_ptr<IECallContext>, cv::gimpl::ie::RequestPool&)>;
```

Comment:
Instead, you could use `using` and make `IECallContext` itself a `std::shared_ptr<>` over some `IECallContextStruct` (just renaming the old structure in a single place).

Reply:
Fixed